sass/_guardians.scss

```diff
@@ -0,0 +1,24 @@
+.guardian {
+    display: flex;
+    flex-direction: row;
+    margin-bottom: 20px;
+
+    .guardian-picture {
+        width: 128px;
+        height: 128px;
+        min-width: 128px;
+        min-height: 128px;
```

Comment:
Why do we set both the `min-` and regular properties?

Reply:
Cargo culting mostly, what's the best practice here?

Reply:
I don't know for sure.